src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -275,7 +286,16 @@ public void start(Job<?, ?> project, boolean newInstance) {
         helper = new Ghprb(this);
 
         if (getUseGitHubHooks()) {
-            if (GhprbTrigger.getDscp().getManageWebhooks()) {
+            //The largest delay is registering hooks with GitHub on startup.
+            //disableRegisterOnStartup system property allows an admin to
+            //disable this behavior.  This assumes hooks are already properly
+            //registered when a job is first created.
+            boolean disable_register_on_startup = SystemProperties.getBoolean(getClass().getName() + ".disableRegisterOnStartup", false);
```

Comment:
I suggest to move this to class level (next to pool definition) and to rename it to `disabletoRegisterOnStartUp`, that seems to be more in line with the rest